server/pbench/bin/pbench-rsync-satellite-process-work-queue

```diff
@@ -61,7 +60,10 @@ def process(todo, options):
         if options.verbose:
             print("os.symlink({}, {})".format(os.path.join(x.archive, x.name), x.link_name))
         if not options.dryrun:
-            os.symlink(os.path.join(x.archive, x.name), x.link_name)
+            try:
+                os.symlink(os.path.join(x.archive, x.name), x.link_name)
+            except OSError:
```

Comment:
Any tarball that triggers an error here is added to the "skipped" list, which is dumped at the end and reported in the status message. I was assuming that we would go back and investigate such failures. The common
case should be that the link exists for some reason: the script could probably deal with that case, but first I'd like to find out how common it is. So I'd like to run it as is for the time being and tweak the handling on the basis of the actual errors we get. Does that seem OK to you?